ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md

```diff
@@ -343,93 +406,109 @@ diem-node --config ~/.0L/vfn.node.yaml  >> ~/.0L/logs/node.log 2>&1
 
 When the sync is ongoing, you'd see something like this:
 
-```
+```log
 ======================================  round is 17897
 ======================================  round is 17898
 ======================================  round is 17899
 ```
-You might see some network errors due to drops, but should again see round numbers. 
 
-This command will tell you the sync state of a RUNNING local node: `db-backup one-shot query node-state`
+You might see some network errors due to drops, but should again see round numbers.
 
+This command will tell you the sync state of a RUNNING local node: `db-backup one-shot query node-state`
 
 > :bangbang: **You must be onboarded by an existing validator to continue**
 >
-> To become a validator a user must display an intention to contribute to the ecosystem. 
+> To become a validator a user must display an intention to contribute to the ecosystem.
 > This can be done by many different ways like building tools, helping out the ecosystem and more.
->  If you would like to contribute reach out the the Hustle Karma channel in [Discord](https://discord.gg/cfXd9Ngk). When a validator is ready to 
+> If you would like to contribute reach out the the Hustle Karma channel in [Discord](https://discord.gg/cfXd9Ngk). When a validator is ready to
 > onboard you they can do it by the following command:
 >
 > ```txs create-validator -u http://[your-ip-address]```
-
-
+>
+> Currently you need 4 validators to vote for you (your onboading partner included), you can kindly ask in the [validator-onboarding discord channel](https://discord.com/channels/833074824447655976/925880684193677332).
 
 ## 7. Restart validator node in `validator` mode
 
-Once the network is in sync and sufficient mining has been done (20 proofs per epoch/day), you are eligible to enter the 
+Once the network is in sync, sufficient mining has been done (20 proofs per epoch/day) and you received the 4 vouches, you are eligible to enter the
```

Comment:
Both validators and miners only need 8 proofs per epoch to be eligible to be in the active set or receive rewards.

Reply:
okay, changed it